src/submission/models.py

```diff
@@ -1991,10 +1991,7 @@ class Meta:
         ordering = ('sequence',)
 
     def __str__(self):
-        return "{} - {}".format(
-            self.pk,
-            self.name,
-        )
+        return self.name
```

Comment:
`Section.name` is nullable, which could lead to this method returning a `NoneType` object. We should return either an empty string, a sentinel value or a string with the section ID in that case

There is also a test case failing with this change.